harness/determined/common/api/_util.py

```diff
@@ -43,6 +45,15 @@ def read_paginated(
         offset = pagination.endIndex
 
 
+def get_default_retry(max_retries: int = 5):
```

Comment:
nit: first impression is to prefer `default_retry` over `get_default_retry`, since there's not really any "getting" going on here.